build.sbt

```diff
@@ -23,7 +23,7 @@ inThisBuild(Def.settings(
   scmInfo := Some(
     ScmInfo(url("https://github.com/akka/akka-http"), "git@github.com:akka/akka-http.git")),
   developers := List(
-    Developer("contributors", "Contributors", "akka-user@googlegroups.com",
+    Developer("contributors", "Contributors", "info@lightbend.com",
```

Comment:
:laughing: I'm not sure that would reach us - but akka-user is obviously wrong as well.

Reply:
akka.official@gmail.com would reach us though